### PR TITLE
Thellimist/des-965 change dUI to look more cloudflarish

### PR DESCRIPTION
**Pull request by thellimist**

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "container.applydefaultsettingscard.button": "Apply",
     "container.applydefaultsettingscard.success": "Your default settings have been successfully set.",
     "container.appNavigation.home": "Home",
-    "container.appNavigation.moresettings": "More Settings",
+    "container.appNavigation.moresettings": "Settings",
```
Comment (thellimist) [inline lang/en.js]: I need to change all MoreSettings to settings if the name change will be accepted

File: src/js/containers/App/App.js [hunk 2]
```diff
@@ -26,34 +26,45 @@ class AppContainer extends Component {
...
-                            { (isLoggedIn() && this.props.state.zoneSettings.entities[this.props.state.activeZone.id]) ? <UnderAttackButton/> : <noscript/> }
-                        </LayoutColumn>
-                    </LayoutRow>
-                    <LayoutRow>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
-                        <LayoutColumn width={1/3}>
-                            <AppNavigation />
-                        </LayoutColumn>
-                        <LayoutColumn width={1/3}>&nbsp;</LayoutColumn>
+                        <header id="header" className="header app-header">
```
Comment (jwineman) [inline src/js/containers/App/App.js]: Is there any css tied to these IDs/classes?

Comment (thellimist) [inline src/js/containers/App/App.js]: There are some css tied to ID's. I tried to remove everything unnecessary.

File: src/js/containers/App/App.js [hunk 3]
```diff
@@ -26,34 +26,45 @@ class AppContainer extends Component {
...
+                                        </div>
+                                    </LayoutColumn>
+                                </div>
+                            </div>
+                        </header>
                     </LayoutRow>
+
+                    <div style={{ marginTop: "-8px" }}>
+                        <LayoutRow>
+                            <div style={{ display: "flex", padding: "1rem", backgroundColor: "#FFF" }}>
```
Comment (thellimist) [inline src/js/containers/App/App.js]: @jwineman I'm using `display: "flex"` here. Would it cause a problem?

File: lang/en.js (hunk 1 above)
Comment (jwineman) [inline lang/en.js]: This is fine.

File: src/js/containers/App/App.js (hunk 3 above)
Comment (jwineman) [inline src/js/containers/App/App.js]: We have [cf-component-flex](https://cloudflare.github.io/cf-ui/#cf-component-flex) for this, we should never be writing `display: "flex";`.  Why doesn't `<LayoutRow>` by itself work here?  Our inline styles should be page / component specific and not structure related.

**Pull request merged**